coalib/settings/Setting.py

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
`Setting: Escape glob characters in parent paths` nonono, you add the glob conversion routines, right? Say that in the commit message :)